settings.py

```diff
@@ -13,7 +13,7 @@
 
 DEBUG = True
 TEMPLATE_DEBUG = DEBUG
-DEBUG_PROPAGATE_EXCEPTIONS = False
+DEBUG_PROPAGATE_EXCEPTIONS = True
```

Comment:
That might have worked then but now it is swallowing the error. I filed https://github.com/mozilla/addons-server/issues/3187 to fix this. I'll take this out.